gnome-initial-setup/pages/timezone/gis-timezone-page.c

```diff
@@ -454,12 +494,18 @@ gis_timezone_page_dispose (GObject *object)
 {
   GisTimezonePage *page = GIS_TIMEZONE_PAGE (object);
   GisTimezonePagePrivate *priv = gis_timezone_page_get_instance_private (page);
+  GNetworkMonitor *monitor = g_network_monitor_get_default ();
 
-  stop_geolocation (page);
+  stop_geolocation_if_needed (page);
 
   g_clear_object (&priv->dtm);
   g_clear_object (&priv->clock);
 
+  if (monitor != NULL && priv->network_monitor_handler_id > 0) {
```

Comment:
Is `monitor` ever going to be `NULL`? The other hunk of this patch assumes it won't.

Reply:
You're right. This was left over after I stopped using the monitor as a private member. I will change it.